Sources/Sound.swift

```diff
@@ -319,6 +335,10 @@ public protocol Player: class {
 
     /// Resume playing.
     func resume()
+    
+    /// Set volume with duration i.e. fade to a new volume over a duration
+    @available(iOS 10.0, *)
```

Comment:
same here with macOS and tvOS
```suggestion
    @available(iOS 10.0, tvOS 10.0, macOS 10.12, *)
```